plugin/xcodebuild.vim

```diff
@@ -3,13 +3,27 @@ command! XTest call <sid>test()
 
 let s:plugin_path = expand('<sfile>:p:h:h')
 
+function! s:bin_script(name)
+  return s:plugin_path . '/bin/' . a:name
+endfunction
+
+function! s:cli_args(...)
+  let cli_args = ''
+
+  for cli_arg in a:000
```

Comment:
Hmm. This doesn't seem to want to work like this. `v:val` isn't defined. :confused: